imports/client/components/SetupPage.tsx

```diff
@@ -482,20 +489,20 @@ const GoogleIntegrationSection = (props: GoogleIntegrationSectionProps) => {
           <li>Template documents must be accessible by the Drive user connected above.</li>
         </ul>
         <GoogleDriveTemplateForm
-          initialSpreadsheetTemplate={props.spreadsheetTemplate}
-          initialDocTemplate={props.docTemplate}
+          initialSpreadsheetTemplate={spreadsheetTemplate}
+          initialDocTemplate={docTemplate}
         />
       </Subsection>
     </Section>
   );
 };
 
 interface EmailConfigFormProps {
-  initialConfig: SettingType | undefined;
+  initialConfig?: SettingType & { name: 'email.branding' };
```

Comment:
I use that elsewhere in this file for variable declarations, rather than interface member declarations. The former doesn't support the `?` syntax (you can't do `const x?: string`).

There are some slight differences in behavior on the assignment side (c.f. [this playground example](https://www.typescriptlang.org/play?#code/JYOwLgpgTgZghgYwgAgILIN4Chm+QDwH4AuZAZzClAHMBuLAXy1ElkRQCFMc99SKqIasgA+yAK4gAJhBigIU+kywIA9iArJIFVAEZS6ALyYG9NRrBaIFDvuRdjGUyvWbtYVACYDyRwVKSMnIgCsjO5m7WYBze9r6Y-hLSsvJSYfRAA)). On the access side, you can run into issues if (e.g.) you're using `Object.keys` or something, but that's less likely to come up for a POJO-style object.